src/loaders/common/callback_server_wrapper.py

```diff
@@ -17,78 +17,101 @@ class Conf:
     """
     Configuration class for the workspace downloader and workspace uploader scripts.
```

Comment:
Should add descriptions of the instance variables that are intended for public use. If any of the instance variables aren't intended for public use, prefix them with `_`

Reply:
Updated!

Reply:
I don't see any changes for this one

Reply:
I added comments for all the instance variables that are intended for public use. There are no private ones.

Reply:
Oh, I see what you're talking about. I mean in the pydoc

Reply:
Is there an example/format I can reference on this?

Reply:
https://github.com/kbase/collections/blob/main/src/service/app_state_data_structures.py#L43-L54

Reply:
Updated!

Reply:
The instance variable count will drop like a rock if we move the downloader specific stuff to the downloader

Reply:
Anyway :+1: